<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditorSettingsWidget</class>
 <widget class="QWidget" name="TextEditorSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FontColors">
      <attribute name="title">
       <string>Font &amp;&amp; Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&amp;Family</string>
              </property>
              <property name="buddy">
               <cstring>fontComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontComboBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&amp;Size</string>
              </property>
              <property name="buddy">
               <cstring>fontSizeSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fontSizeSpinBox">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="value">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="antialiasCheckBox">
              <property name="text">
               <string>Antialias</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>&amp;Zoom</string>
              </property>
              <property name="buddy">
               <cstring>zoomSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="zoomSpinBox">
              <property name="suffix">
               <string notr="true" comment="%" extracomment="%"/>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>600</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Behavior">
      <attribute name="title">
       <string>Behavior</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Tabs &amp;&amp; Indentation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="tabsToSpacesCheckBox">
            <property name="text">
             <string>Insert &amp;spaces insteed of tabs</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Ta&amp;b size</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>tabSizeSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="tabSizeSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="autoIndentCheckBox">
            <property name="text">
             <string>Enable automatic &amp;indentation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&amp;Indent size</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>indentSizeSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="indentSizeSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="backspaceIndentCheckBox">
            <property name="text">
             <string>&amp;Backspace follows indentation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Mouse</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="mouseNavigationCheckBox">
            <property name="text">
             <string>Enable &amp;mouse navigation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mouseZoomingcheckBox">
            <property name="text">
             <string>Enable scroll &amp;wheel zooming</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="activateOnMouseEnterCheckBox">
            <property name="text">
             <string>&amp;Activate document on mouse Enter event</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Display">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Text Wrapping</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="textWrappingCheckBox">
            <property name="text">
             <string>Enable text &amp;wrapping</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="showRightMarginCheckBox">
              <property name="text">
               <string>Show right &amp;margin at column:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="rightMarginColumnSpinBox">
              <property name="minimum">
               <number>40</number>
              </property>
              <property name="maximum">
               <number>600</number>
              </property>
              <property name="value">
               <number>120</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="showLineNumbersCheckBox">
            <property name="text">
             <string>Show line &amp;numbers</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="centerCursorOnScrollCheckBox">
            <property name="text">
             <string>Center &amp;cursor on scroll</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="showFoldingMarkersCheckBox">
            <property name="text">
             <string>Show &amp;foding markers</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="highlightCurrentLineCheckBox">
            <property name="text">
             <string>Highlight current &amp;line</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="markTextChangesCheckBox">
            <property name="text">
             <string>Mark &amp;text changes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="highlightBlocksCheckBox">
            <property name="text">
             <string>Highlight &amp;blocks</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="showWitespacesCheckBox">
            <property name="text">
             <string>Show &amp;whitespaces</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="animateMatchingParenthesesCheckBox">
            <property name="text">
             <string>&amp;Animate matching parentheses</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>117</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
